tools/JsonCli/src/Program.cs

```diff
@@ -144,7 +144,7 @@ private static int Main(string[] args)
                 TemplateCreationResult result = _templateCreator.InstantiateAsync(info, null, null, null, templateArgs, true, false, null).Result;
                 if (result.Status == CreationResultStatus.Success)
                 {
-                    PostActionDispatcher postActionDispatcher = new PostActionDispatcher(EnvironmentSettings, result, AllowPostActionsSetting.Yes, false);
+                    PostActionDispatcher postActionDispatcher = new PostActionDispatcher(EnvironmentSettings, new New3Callbacks(), result, AllowPostActionsSetting.Yes, false);
```

Comment:
What is `New3Callbacks`?  I don't see it defined or imported anywhere.  Is it just a C Sharp thing?  I couldn't find it in a google search

Reply:
I mean it's a part of one of the namespaces at the top of this file (probably "Microsoft.TemplateEngine.Cli"). Since I updated the package references, the build was failing because `PostActionDispatcher` now needed me to pass a `New3Callbacks` in